perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_resource_attribute_def_def_redmineProjectID.java

```diff
@@ -0,0 +1,51 @@
+package cz.metacentrum.perun.core.impl.modules.attributes;
+
+import cz.metacentrum.perun.core.api.Attribute;
+import cz.metacentrum.perun.core.api.AttributeDefinition;
+import cz.metacentrum.perun.core.api.AttributesManager;
+import cz.metacentrum.perun.core.api.Resource;
+import cz.metacentrum.perun.core.api.exceptions.InternalErrorException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeAssignmentException;
+import cz.metacentrum.perun.core.api.exceptions.WrongAttributeValueException;
+import cz.metacentrum.perun.core.api.exceptions.WrongReferenceAttributeValueException;
+import cz.metacentrum.perun.core.impl.PerunSessionImpl;
+import cz.metacentrum.perun.core.implApi.modules.attributes.ResourceAttributesModuleAbstract;
+import cz.metacentrum.perun.core.implApi.modules.attributes.ResourceAttributesModuleImplApi;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+/**
+ * Module for redmine project ID
+ *
+ * @author Peter Balcirak <peter.balcirak@gmail.com>
+ * @date 29.3.2016
+ */
+public class urn_perun_resource_attribute_def_def_redmineProjectID extends ResourceAttributesModuleAbstract implements ResourceAttributesModuleImplApi {
+
+	private static final Pattern pattern = Pattern.compile("^[a-z][-_a-z0-9]+$");
+
+	@Override
+	public void checkAttributeValue(PerunSessionImpl perunSession, Resource resource, Attribute attribute) throws InternalErrorException, WrongAttributeValueException, WrongReferenceAttributeValueException, WrongAttributeAssignmentException {
+		String id = (String) attribute.getValue();
+		if (id == null) {
+			throw new WrongAttributeValueException(attribute, resource, "Attribute can't be empty.");
```

Comment:
Please add suggestion what attribute value is correct to error message. Something like you have few line later is suitable.